NaReguaApp/src/routes/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { View, ActivityIndicator } from 'react-native';
+
+import { useAuth } from '../contexts/auth';
```

Comment:
Codacy has a fix for the issue: [Replace `·useAuth·` with `useAuth`](https://app.codacy.com/gh/UFAPE-Projetao20202/naregua_front/pullRequest?prid=8386487)
```suggestion
import {useAuth} from '../contexts/auth';
```